recipes/example/meta.yaml

```diff
@@ -1,34 +1,44 @@
-{% set version = "1.0" %}
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "3.8.2" %}
 
 package:
-  name: example
-  version: {{ version }}
+    name: simplejson
+    version: {{ version }}
 
 source:
-  git_url: https://github.com/conda-forge/staged-recipes.git
-  git_tag: 24507f6c6b2d2e03caee550e13830df15d791751
+    # It is preferable to get the package "closer" to the source. For example use GitHub instead PyPI whenever possible.
+    fn: simplejson-{{ version }}.tar.gz
+    url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+    # md5 is OK but sha256 is desirable.
+    sha256: 0041407316fbd35786883436c9ffaba4b799569174f22ee54e758f72249b1c67
 
 build:
-  number: 0
+    number: 0
```

Comment:
I don't want to add too much text here. Hopefully it is clear enough that recipes must be build `0` when entering staged-recipes. (Otherwise we won't merge it anyways as it already exists as a feedstock :wink:)